src/utils/download.ts

```diff
@@ -52,3 +54,16 @@ export async function downloadProject(project: Project, uri?: string) {
     document.body.removeChild(link);
   });
 }
+
+async function addDevKey(queue: Array<{filePrefix: string; file: File}>, uri: string) {
+  const app = (window as any).app;
+  const contractName = `studio-${uri}`;
+  const keyPair = await app.state.keyStore.getKey(contractName);
+  if (keyPair && keyPair.getPublicKey() && keyPair.getSecretKey()) {
```

Comment:
can we just make it a contract for key stores that `publicKey` / `secretKey` shouldn't be null?

Reply:
sure, removed extra checks